erica/application/EricaAuftrag/EricaAuftrag.py

```diff
@@ -11,7 +11,7 @@ class BasePayloadDto(BaseModel):
 
 
 class EricaAuftragDto(BaseModel):
```

Comment:
We wanted to add an additional class attribute for the "result"?

Reply:
This is part of the other pr. But we could add it here, too.

Reply:
Ok